UIComponents/Sources/UIComponents/Profile/CalendarView.swift

```diff
@@ -0,0 +1,110 @@
+import UIKit
+import Style
+
+public extension CalendarView {
+    struct Model {
+        public let description: String
+        public let textFieldText: String
+        public let textFieldFactory: TextFieldGenerating
+
+        public init(
+            description: String,
+            textFieldText: String,
+            textFieldFactory: TextFieldGenerating = TextFieldFactory()
+        ) {
+            self.description = description
+            self.textFieldText = textFieldText
+            self.textFieldFactory = textFieldFactory
+        }
+    }
+}
+
+public class CalendarView: UIView {
```

Comment:
We might have an another view that will need to use this name because it's will be appropriate in that case. When I see it, it seems like a custom calendar. 

`CalendarView` -> `DateTextFieldView`